index.html

```diff
@@ -37,7 +37,160 @@ <h1>I'm Newton <br>Glad to see you!</h1>
+                        <li>
+                            css
+                        </li>
+                        <li>
+                            javascript
+                        </li>
+                    </ul>
+                    <a href="" class="project-links">See Project</a>
+                </div>
+            </li>
+            <li class="categories">
+                <img src="./img/portfolio-tonic2.png" alt="Tonic Category Image">
+                <div class="left-block">
+                    <h2 class="project-title">
+                        Tonic
+                    </h2>
+                    <span class="canopy">
+                        CANOPY
+                        <ul>
+                            <li><img src="./img/counter.png" alt=""></li>
+                            <li>Back End Dev</li>
+                            <li><img src="./img/counter.png" alt=""></li>
+                            <li>2015</li>
+                        </ul>
+                    </span>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="portfolio-tags">
+                        <li>
+                            html
+                        </li>
+                        <li>
+                            css
+                        </li>
+                        <li>
+                            javascript
+                        </li>
+                    </ul>
+                    <a href="" class="project-links">See Project</a>
+                </div>
+            </li>
+            <li class="categories">
+                <img src="./img/portfolio-stories2.png" alt="Tonic Category Image">
+                <div class="left-block">
+                    <h2 class="project-title">
+                        Multi-Post Stories
+                    </h2>
+                    <span class="canopy">
+                        CANOPY
+                        <ul>
+                            <li><img src="./img/counter.png" alt=""></li>
+                            <li>Back End Dev</li>
+                            <li><img src="./img/counter.png" alt=""></li>
+                            <li>2015</li>
+                        </ul>
+                    </span>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="portfolio-tags">
+                        <li>
+                            html
+                        </li>
+                        <li>
+                            css
+                        </li>
+                        <li>
+                            javascript
+                        </li>
+                    </ul>
+                    <a href="" class="project-links">See Project</a>
+                </div>
+            </li>
+        </ul>
+    </section>
+    <section class="about-myself">
+        <h1>About Myself</h1>
```

Comment:
Your design 👇 (No border-radius)
![image](https://user-images.githubusercontent.com/78251927/205412317-1d33556d-ae14-4d3c-958c-b5d9389cadd7.png)

Figma design 👇 (Section has border-radius)
![image](https://user-images.githubusercontent.com/78251927/205412421-4fe3bb16-293d-43b4-940a-d83dc9098d19.png)

- [ ] Please, let us include the `border-radius` as shown above ☝️